SwiftyJSONAPI/JSONAPIError.swift

```diff
@@ -9,32 +9,32 @@
 import Foundation
 
 
-public class JSONAPIError: JSONPrinter {
+open class JSONAPIError: JSONPrinter {
```

Comment:
I think you are right, in some cases it probably doesn't make sense to make a class or property open, but instead should remain public. Any reason why we are making everything open @billyto ?